tslint.json

```diff
@@ -27,7 +27,7 @@
     ],
     "no-arg": true,
     "no-bitwise": true,
-    "no-console": [true, "debug", "info", "time", "timeEnd", "trace"],
+    "no-console": true,
```

Comment:
We're not using tslint. I think this file can be removed. Configure .eslintrc instead.

Reply:
👍🏻